data/sidebars/tutorials.yml

```diff
@@ -3,6 +3,11 @@
     - title: Introduction
       link: /tutorials/introduction/
 
+- group: Basics
```

Comment:
I think it makes sense not creating a new group for that, but rather rename the _Overview_ group to _Basics_. Cause I suspect we won't fill the current _Overview_ group with much more. Otherwise we could change later easily, since groups are not attached to urls

Reply:
Done.